### ISSUE TITLE
Add focusability to a11y of textfield

### ISSUE THREAD

**Issue by goderbauer** [flutter/flutter#44742]
The Textfield widget should provide information about its focusability to the semantics tree.

/cc @gspencergoog

**Comment by Piinks** [comment]
@chunhtai @justinmc is this old issue resolved now?

**Comment by chunhtai** [comment]
looks like textfield is missing isFocusable flag.

**Comment by chunhtai** [comment]
cc @Renzo-Olivares

**Comment by Renzo-Olivares** [comment]
Closing, this was fixed by https://github.com/flutter/flutter/pull/173235. Please feel free to re-open if this was not the case.
